### ISSUE TITLE
Comment Admin doesn't show name/email for logged-in user comments

### ISSUE THREAD

**Issue by pjayme** [silverstripe/silverstripe-comments#260]
Comments made by logged in users don't show the user's name or email address in the admin screen. This makes it hard to determine who has made comments across the site.

It looks like changing the `$summary_fields` on `Comment` should resolve this - there's already a `getAuthorName()` method that can be used, but we'd need an equivalent `getAuthorEmail()` method.

**Comment by pjayme** [comment]
SilverStripe: 4.1
Comments: 3.1.2

Alright cool, will go for DataExtension approach.

Cheers

**Comment by robbieaverill** [comment]
>Comments made by logged in users don't show the user's name or email address in the admin screen. 

If this is the case, it's definitely a bug. Should look like this (anonymously posted comments):

![image](https://user-images.githubusercontent.com/5170590/42073909-626c0fc2-7bbd-11e8-8008-b1ae39f80256.png)

**Comment by pjayme** [comment]
Yup, this is definitely the case, it doesn't display for logged in users - we are using it for an intranet so all our users will be logged in by default. We wouldn't get anonymous contents, it is also reinforced using the `require_login: true` permission.

**Comment by pjayme** [comment]
Have done something like this on my `CommentExtension.php` for the time being:

```
<?php

namespace MySite\Extensions;

use SilverStripe\ORM\DataExtension;

class CommentExtension extends DataExtension
{
    /**
     * @var array
     */
    private static $summary_fields = [
        'Author.FirstName' => 'Author name',
        'Author.Email' => 'Author email',
    ];

    public function updateSummaryFields(&$fields)
    {
        $fields['Name'] = '';
        $fields['Email'] = '';
    }
}
```

![screen shot 2018-06-29 at 5 04 59 pm](https://user-images.githubusercontent.com/27745093/42074124-9af678d6-7bbe-11e8-9a5a-cc39e3054287.png)

**Comment by pjayme** [comment]
Also another thing which might be a UX enhancement but the `Comment` label for its column isn't the most suitable as some of those entries are `Replies` so its quite tricky for a user to see which is a `Comment` and which is a `Reply` to a comment.

Perhaps the label should be `Comment/Replies`?

**Comment by robbieaverill** [comment]
Fixed with #273, will be available in the next CWP recipe release
